hoomd/logging.py

```diff
@@ -135,14 +135,16 @@ class _NamespaceFilter:
     """Filter for creating the proper namespace for logging object properties.
 
     Attributes:
-        remove_names (set[str]): A set of names which to remove for the logging
-            namespace whenever encountered.
-        base_names (set[str]): A set of names which indicate that the next
-            encountered name in the string should be skipped. For example, if a
-            module hierarchy went like ``project.foo.bar.Bar`` and ``foo``
+        remove_names (set[str], optional): A set of names which to remove for
+            the logging namespace whenever encountered.
+        base_names (set[str], optional): A set of names which indicate that the
+            next encountered name in the string should be skipped. For example,
+            if a module hierarchy went like ``project.foo.bar.Bar`` and ``foo``
```

Comment:
```suggestion
            if a module hierarchy was structured as ``project.foo.bar.Bar`` and ``foo``
```

Reply:
Nope not using pytest, or a lot of custom code to test this behavior. I could test  `_NamespaceFilter` by itself.